debian/docker-entrypoint.sh

```diff
@@ -1,6 +1,30 @@
 #!/bin/bash
 set -eu
 
+# usage: file_env VAR [DEFAULT]
+#    ie: file_env 'XYZ_DB_PASSWORD' 'example'
+# (will allow for "$XYZ_DB_PASSWORD_FILE" to fill in the value of
+#  "$XYZ_DB_PASSWORD" from a file, especially for Docker's secrets feature)
+file_env() {
+	local var="$1"
+	local fileVar="${var}_FILE"
+	local def="${2:-}"
+	if [ "${!var:-}" ] && [ "${!fileVar:-}" ]; then
+		echo >&2 "error: both $var and $fileVar are set (but are exclusive)"
+		exit 1
+	fi
+	local val="$def"
+	if [ "${!var:-}" ]; then
+		val="${!var}"
+	elif [ "${!fileVar:-}" ]; then
+		val="$(< "${!fileVar}")"
+	fi
+	export "$var"="$val"
+	unset "$fileVar"
+}
+
+file_env 'RABBITMQ_DEFAULT_PASS'
```

Comment:
Yeah, that'd be my preference.  Alternatively, we could simply re-use the arrays we've got and add this for _all_ variables, but then we'll technically support several `XYZ_FILE_FILE` variables, which is a tad on the nutty side (so maybe later we can consider doing it for all vars which don't match `*_FILE` already, but that's definitely something I'm fine waiting on until someone asks for more variables to be supported). :smile: :+1: